src/panel/poi/PoiBlockContainer.jsx

```diff
@@ -33,6 +35,7 @@ export default class PoiBlockContainer extends React.Component {
       {hourBlock && <HourBlock block={hourBlock} covid19enabled={!!displayCovidInfo} />}
       {informationBlock && <InformationBlock block={informationBlock} />}
       {websiteBlock && <WebsiteBlock block={websiteBlock} poi={this.props.poi} />}
+      {phoneBlock && <PhoneBlock block={phoneBlock} />}
```

Comment:
The phone block should be put before the website block, according to the mockup.

Reply:
ok done